app/bundles/EmailBundle/Model/SendEmailToContact.php

```diff
@@ -249,8 +246,6 @@ public function reset()
         $this->statBatchCounter  = 0;
         $this->contact           = [];
 
-        $this->dncModel->clearEntities();
```

Comment:
It would be nice to replace this too instead of removing it but it's not possible because we'd need the DNC entities to do it, right? Just checking.

Reply:
It would not be possible because we do not have dnc entities here.

Reply:
I thought so too. I hope this won't cause any memory leaks.

Reply:
This should be done in `\Mautic\LeadBundle\Model\DoNotContact::addDncForContact`